docs/_content/standaard/standaard.md

```diff
@@ -316,6 +316,76 @@ tegen het `Zaaktype.productenOfDiensten` van het betreffende zaaktype. De
 diensten op het zaaktype zijn.
 
+#### Archiveren
+
+**Afleiden van archiveringsparameters**
+
+Het resultaat van een zaak is bepalend voor het archiefregime. Bij het zetten
+van het resultaat van een zaak MOETEN de attributen `Zaak.archiefnominatie`
+en `Zaak.archiefactiedatum` bepaald worden als volgt:
+
+1. Indien de zaak geen `archiefnominatie` heeft, dan MOET deze overgenomen
+   worden uit `Resultaat.Resultaattype.archiefnominatie`
+2. Indien `Resultaat.Resultaattype.archiefactietermijn` gevuld is:
+    1. Bepaal de `brondatum` van de archiefprocedure
+        1. Consulteer het groepattribuut `Resultaat.Resultaattype.brondatumArchiefprocedure`
+        2. Afhankelijk van de waarde van `afleidingswijze`:
+            * `afgehandeld` -> gebruik `Zaak.einddatum`
+            * `hoofdzaak` -> gebruik `Zaak.hoofdzaak.einddatum`
+            * `eigenschap` -> gebruik de waarde van de eigenschap met als naam
+              de waarde van
+              `Resultaat.Resultaattype.brondatumArchiefprocedure.datumkenmerk`
+            * `ander_datumkenmerk` -> brondatum MOET handmatig afgeleid en
+              gezet worden
+            * `zaakobject` -> zoek de gerelateerde objecten van type
+              `Resultaat.Resultaattype.brondatumArchiefprocedure.objecttype`.
+              Lees van elk object het attribuut met de naam
+              `Resultaat.Resultaattype.brondatumArchiefprocedure.datumkenmerk`
+              en gebruik de maximale waarde.
+            * `termijn` -> `Zaak.einddatum` + `Resultaat.Resultaattype.brondatumArchiefprocedure.procestermijn`
+            * `gerelateerde_zaak` -> TODO
+            * `ingangsdatum_besluit` -> maximale `Besluit.ingangsdatum` van alle
+              gerelateerde besbluiten
+            * `vervaldatum_besluit` -> maximale `Besluit.vervaldatum` van alle
+              gerelateerde besbluiten
+    2. Zet `Zaak.archiefactiedatum` als `brondatum + Resultaat.Resultaattype.archiefactietermijn`
+
+Indien de archiefactiedatum niet bepaald kan worden, dan MAG er geen datum
+gezet worden. Dit kan voorkomen als de brondatum niet bepaald kan worden of
+de archiefactietermijn niet beschikbaar is.
+
+**Zetten `Zaak.archiefstatus`**
+
+De standaardwaarde voor archiefstatus is `nog_te_archiveren`. Indien een andere
+waarde gezet worddt, dan MOETEN alle gerelateerde informatieobjecten de status
+`gearchiveerd` hebben.
+
+De attributen `Zaak.archiefnominatie` en `Zaak.archiefactiedatum` MOETEN een
+waarde krijgen als de de archiefstatus een waarde krijgt anders dan
+`nog_te_archiveren`.
+
+Indien deze voorwaarden niet voldaan zijn, dan MOET het ZRC met een `HTTP 400`
+foutbericht antwoorden.
+
+**Vernietigen van zaken**
+
+Bij `DELETE` requests op zaken MOETEN de zaak en gerelateerde objecten fysiek
+uit de opslag verwijderd worden. Soft-deletes zijn NIET TOEGESTAAN. Onder
+gerelateerde objecten wordt begrepen:
+
+- `zaak` - de deelzaken van de verwijderde hoofzaak
```

Comment:
Daar zijn twee strategieen voor: (1) hoofdzaken en deelzaken volgens de langste bewaartermijn bewaren en dan tesamen vernietigen (2) losgekoppeld, dus volgens eigen bewaartermijnen bewaren, en apart vernietigen. Daar moeten we nog een keus in maken.

Reply:
de database cascade is nu erg duidelijk - indien een zaak deelzaken heeft, dan heeft een delete actie op de hoofdzaak ook delete op de deelzaken tot gevolg

Reply:
Voor nu goed. Maar kan t.z.t. nog veranderen, of zelfs per zaaktype worden ingesteld. Hebben we hulp bij nodig van experts.